### PR TITLE
<AnalyticsSummaryCard/> - new component

### PR DESCRIPTION
**Pull request by justame**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6312

File: src/AnalyticsSummaryCard/AnalyticsSummaryCard.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  isPercentageVisible: PropTypes.bool,
+  percentageTooltip: PropTypes.string,
+};
+
+AnalyticsSummaryCard.defaultProps = {
+  isLoading: false,
+  ctaButton: null,
+  footer: null,
+  onChartHover: noop,
+  isPercentageVisible: true,
```
Comment (EgoziE) [inline src/AnalyticsSummaryCard/AnalyticsSummaryCard.js:161]: We don't allow boolean props to be set as true by default.
You can rename this prop to "hidePercentage" and make it false instead.

File: src/AnalyticsSummaryCard/AnalyticsSummaryCard.st.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  min-width: 216px;
+}
+
+.root.hovered {
+  background-color: #f4fafe;
+}
+
+.root.clickable {
+  cursor: pointer;
+}
```
Comment (EgoziE) [inline src/AnalyticsSummaryCard/AnalyticsSummaryCard.st.css:19]: These should be changed to Stylable states.
Take a look at [Stylable Pseudo Classes](https://stylable.io/docs/references/pseudo-classes)

File: src/AnalyticsSummaryCard/AnalyticsSummaryCard.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        </div>
+        <div></div>
+      </div>
+    );
+  }
+}
+
+AnalyticsSummaryCard.displayName = 'AnalyticsSummaryCard';
+
+AnalyticsSummaryCard.propTypes = {
```
Comment (EgoziE) [inline src/AnalyticsSummaryCard/AnalyticsSummaryCard.js:128]: Don't forget to add a comment before each propType, this is important for automatic documentation

**Review by EgoziE — CHANGES_REQUESTED** [review]

File: src/AnalyticsSummaryCard/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import IconButton from '../IconButton';
```
Comment (EgoziE) [inline src/AnalyticsSummaryCard/index.d.ts:2]: Are you going to use IconButton?

**Comment by EgoziE** [comment]
@milkyfruit ready for UX review

File: src/AnalyticsSummaryCard/index.d.ts (hunk 4 above)
Comment (justame) [reply src/AnalyticsSummaryCard/index.d.ts:2]: I should remove it

**Comment by assafzi** [comment]
> @milkyfruit ready for UX review

UX is here.
The component looks good.
I have some questions/improvements

- [ ] Icon Button's skin should be Inverted
`<IconButton skin="inverted"> 
`

- [ ] Sparkline should have tooltip on hover as described in the component's page
https://snipboard.io/R3Md4V.jpg

- [ ] Title and value tooltip - when the field is empty there's no need to display a tooltip
https://snipboard.io/ahRSW4.jpg

- [ ] isTrendVisible prop should be next to the Trend prop in the playground (currently is on misc props). It will make more sense
https://snipboard.io/wKpImy.jpg

- [ ] Why invertedTrend prop is needed?
https://snipboard.io/5gCX01.jpg

- [ ] Need to add mouse and keyboard a11y as described in the spec
https://zpl.io/adl4PWp


I'm working on better examples in the description tab. Will add it tomorrow.

**Comment by assafzi** [comment]
Following previous comment, here's a better description examples

- [ ] **Simple Usage**
A simple example.
image


- [ ] **Footer**
Summary card can have a footer for extra content.
The footer is a node box with no height limitation.
image

- [ ] **Trend**
The trend indicates the change of the value in percentage.
It has 2 modes:
Positive - colored green with arrow upwards
Negative - colored red with arrow downwards.
 If there’s no change in value, trend won’t be displayed.
image

- [ ] **Action Button**
Summary card can have action button inside to assist with any action needed, since it’s a node.
image

- [ ] **Clickable Card**
Card can be clickable or not.
image

- [ ] **Loader**
Card supports loading state. It will replace the action button when it’s active. If the card is clickable it won’t be as long the loader is active. 
image

- [ ] **Sparkline Width**
Sparkline width is set by px only. The minimum width is 69px.
image

- [ ] **Tooltip On Title**
If needed, the title can have tooltip to better explain the context.
image

- [ ] **Tooltip On Value**
When value appears as abbreviated number, tooltip will appear on hover.
image

Also, in the API, in the onClick, please change the description of it to indicate clearer that it controls the clickable on card.
- [ ]  Card would be clickable

**Comment by assafzi** [comment]
UX approve. Thanks.

**Pull request merged**